src/content/de/announcements/2025-09-26.md

```diff
@@ -0,0 +1,129 @@
+---
+title: 'Sovereign Cloud Stack R9 freigegeben'
+authors:
+    - name: 'Sovereign Cloud Stack'
+postType: announcements
+language: de
+date: '2025-09-26'
+slug: release9
+template: 'post'
+---
+
+## Sovereign Cloud Stack R9 kommt mit breiteren Nutzungsmöglichkeiten
+
+**Europa, 2025-09-26: Sovereign Cloud Stack Version 9 ist fertig**
+
+Der inspirierende und ausverkaufte SCS Summit 2025 in Berlin hat erneut
+deutlich gezeigt, dass es ein andauerndes Interesse an der Nutzung der
+SCS-Standards und der Software gibt, um damit Abhängigkeiten und Risiken
+bei der Umsetzung digitaler Lösungen zu reduzieren, sowohl aus dem
+öffentlichen Sektor als auch aus der Industrie. Für viele Teilnehmer:innen
+war es überhaupt keine Frage, dass wir zusammen offene Standards und
+offene Software bauen müssen um in der Digitalisierung voran zu kommen
+und dabei die Kontrolle, Innovations- und Wertschöpfungsfähigkeit erhalten.
+
+Die SCS Community erstellt einen modularen Open Source Software Stack, welcher
+die SCS Standards umsetzt, und der in Teilen oder als Ganzes von über einem
+halben Dutzend Anbieter genutzt wird, um öffentliche oder private Clouddienste
+anzubieten. Die Technologie kann sicherlich als bewährt angesehen werden, nachdem
+sie seit Jahren der Motor für öffentliche Clouds ist, ohne dass es zu ungeplanten
+Ausfällen kam und zum Beispiel auch den Unterbau für die "Bayern Cloud Schule"
```

Comment:
```suggestion
Ausfällen kam und zum Beispiel auch den Unterbau für die "BayernCloud Schule"
```

see: https://www.bayern.de/datensicher-praxistauglich-und-zukunftsorientiert-die-bayerncloud-schule-startet-bayernweit-in-den-regelbetrieb-mit-dem-uebergang-in-den-regelbetrieb-ist-die-bayerncloud-schule-bycs-fla/